inc/snowball-editor.php

```diff
@@ -122,8 +150,8 @@
     'templates'   => $templates,
     'names'       => $names,
     'adminUrl'    => admin_url(),
+    'pluginsUrl'  => plugins_url("snowball"),
     'includesUrl' => includes_url(),
-    'pluginsUrl'  => plugins_url('', dirname(__FILE__)),
```

Comment:
Looks like you didn't fetch the latest `develop` before rebasing. I changed this in an earlier commit to get rid of this hardcoded path we missed.